woocommerce/wc-admin
# Add Inbox Note example code to docs

Partially fixes #7935

This PR adds an Inbox Note example in docs.

Credits to @loranallensmith for original inbox note example extension from https://github.com/loranallensmith/admin-note-example/blob/admin-note-tutorial/admin-note-example.php

### Screenshots

<img src="https://user-images.githubusercontent.com/3747241/145206583-478b52b2-1a60-4858-9996-1f0fbbbbf1cf.png" width="450">

### Detailed test instructions:

Since it's a documentation update, you should review the code written.

In order to the example, you need to test it in a local development repository.

- In your terminal, run `npm run example -- --ext=simple-inbox-note`
- Observe that webpack is able to emit `../../../../simple-inbox-note/woocommerce-admin-simple-inbox-note-example.php`
- Go to WordPress > Installed Plugins, look for `WooCommerce Admin Simple Inbox Note Example` plugin
- Activate the plugin
- Go to WooCommerce > Home
- Observe the note "Hello From Inbox Note!" exists

No changelog
